core/docs/Cargo.toml

```diff
@@ -6,9 +6,19 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+# Explanation: You want the option to remove configuration docs if you distribute e.g. a `deb`, but want the person who just cloned the repo to be able to see at a glance what kinds of features are accessible to them, what they do and how to use them. #1975 should be fixed soon. 
```

Comment:
If we had some [Build Scripts](https://doc.rust-lang.org/cargo/reference/build-scripts.html) to generate the doc with the build, the features would be synchronized between the docs and the distribution

Reply:
Could you add that to the issue?